boot/makebootable.go

```diff
@@ -155,33 +162,14 @@ func makeBootable16(model *asserts.Model, rootdir string, bootWith *BootableSet)
 	return nil
 }
 
-func makeBootable20(rootdir string, bootWith *BootableSet, bootFlags []string) error {
-	// we can only make a single recovery system bootable right now
-	recoverySystems, err := filepath.Glob(filepath.Join(rootdir, "systems/*"))
-	if err != nil {
-		return fmt.Errorf("cannot validate recovery systems: %v", err)
-	}
-	if len(recoverySystems) > 1 {
-		return fmt.Errorf("cannot make multiple recovery systems bootable yet")
-	}
-
-	if bootWith.RecoverySystemLabel == "" {
-		return fmt.Errorf("internal error: recovery system label unset")
-	}
-
+func configureBootloader(rootdir string, bootWith *BootableSet, bootMode string, opts *bootloader.Options, bootFlags []string) error {
```

Comment:
nitpick: opts should come directly after rootdir to match MakeBootablePartition

Reply:
Done